src/display/editor/highlight.js

```diff
@@ -530,6 +559,8 @@ class HighlightEditor extends AnnotationEditor {
     const div = super.render();
     if (this.#isFreeHighlight) {
       div.classList.add("free");
+    } else {
+      this.div.addEventListener("keydown", this.#boundKeydown);
```

Comment:
This listener isn't removed anywhere, but I'm assuming that's actually *on purpose* in this case?

Reply:
Why should we remove it ?

Reply:
Recently I've reviewed a couple of PRs where some event listeners were accidentally not being removed as expected, hence I've simply amended by PR review "checklist" to look closer at any listeners :-)